src/Behat/Gherkin/Parser.php

```diff
@@ -565,7 +566,16 @@ protected function parseExamples()
      */
     protected function parseTable()
     {
-        return new TableNode($this->parseTableRows());
+        try {
+            return new TableNode($this->parseTableRows());
```

Comment:
I would probably keep the `parseTableRows` outside the `try/catch` so that we catch only the exceptions from `TableNode`.